gradoop-flink/src/test/java/org/gradoop/flink/model/impl/functions/epgm/filters/CombineableFilterTest.java

```diff
@@ -0,0 +1,37 @@
+package org.gradoop.flink.model.impl.functions.epgm.filters;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class CombineableFilterTest {
```

Comment:
Please add an integration test for apply a (complex) combined filter to a dataset and check if the result is equal to the expected. There are some test graphs in the gradoop-flink test resources you can use.